src/App.js

```diff
@@ -1,73 +0,0 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-import axios from 'axios'
-import AnimalList from './Components/AnimalList'
-import AddAnimal from './Components/AddAnimal';
-
-const BASE_URL = 'http://localhost:4000'
-
-class App extends Component {
-  constructor(){
-    super();
-    this.state = {
-      animalList : []
-    }
-  }
-  componentDidMount(){
-    axios({
-      method: 'GET',
-      url: BASE_URL + '/api/animals'
-    }).then(response => {
-      this.setState({animalList: response.data})
-    })
-    console.log('component did mount')
-    console.log(this.state.animalList);
```

Comment:
just want you to be aware - logging out `this.state.animalList` here might create some unexpected logs, this log will run before the `setState` on line 22 will run, so you'll be logging out a _previous_ version of `this.state.animalList`